substrate/client/tracing/src/logging/mod.rs

```diff
@@ -652,36 +654,4 @@ mod tests {
 			assert!(stderr.contains(&line));
 		}
 	}
-
-	#[test]
-	fn control_characters_are_always_stripped_out_from_the_log_messages() {
```

Comment:
The case is impossible now, so the test is no longer needed.

Reply:
@bkchr If we only use `console` to style strings, when it renders ANSI strings, it will check the global `colors_enabled()` and `colors_enabled_stderr()`, if they're `false`, it will skip control codes.

So we don't need to strip ANSI control codes because they will not be rendered if colors diasbled

Reply:
The only restriction is we must style strings using `console` only, it doesn't take care of other sources of ANSI control codes

Reply:
For this case, `console` won't sanitize the `RAW_LINE`.
But if we only use `console`, it is impossible to get a string with ANSI control codes.